woeip/templates/includes/_navigation.html

```diff
@@ -10,13 +10,13 @@
   </div>
   <div class="top-bar-right">
     <ul class="dropdown vertical medium-horizontal menu" >
-      <li ><a class="button {{ upload_page }}" tabindex="0" aria-label="add data" href="{% url 'upload' %}">Add Data</a></li>
-      <li><a class="button" tabindex="0" aria-label="view maps" href="#">View Maps</a></li>
-      <li>
+      <li ><a class="button {{ upload_page }}" tabindex="0" aria-label="add data" href="{% url 'upload' %}">Add data</a></li>
+      <li><a class="button" tabindex="0" aria-label="view maps" href="#">View maps</a></li>
       {% if user.is_authenticated %}
-        <a class="button" tabindex="0" aria-label="sign out" href="{% url 'logout' %}">Sign out</a>
+        <li><a class="button" tabindex="0" aria-label="sign out" href="{% url 'password_change' %}">Change password</a></li>
```

Comment:
`aria-label` should be "password_change". We should also add aria-labels to other buttons - I'll make it an issue.